ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -34,17 +43,24 @@ public partial class UIManagerModule : ReactContextNativeModuleBase, ILifecycleE
         /// <param name="reactContext">The React context.</param>
         /// <param name="viewManagers">The view managers.</param>
         /// <param name="uiImplementation">The UI implementation.</param>
+        /// <param name="window">The ApplicationView/Framework Element.</param>
         public UIManagerModule(
             ReactContext reactContext,
             IReadOnlyList<IViewManager> viewManagers,
-            UIImplementation uiImplementation)
+            UIImplementation uiImplementation,
+#if WINDOWS_UWP
+            ApplicationView window)
+#else
+            FrameworkElement window)
+#endif
```

Comment:
This is the most centralized way to decouple this class from the concrete type, and eliminate a global instance access. This approach will allow for parallelization of all tests being run without the worry of conflicts in global mutable state.